### PR TITLE
Add adorn from PyPI

### PR DESCRIPTION
**Pull request by jameswinegar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jacobBaumbach** [comment]
I am willing to be a maintainer.

**Comment by jameswinegar** [comment]
```
2021-12-11T18:17:28.9740016Z Packaging adorn
2021-12-11T18:17:28.9741220Z INFO conda_build.build:build(2289): Packaging adorn
2021-12-11T18:17:29.2273167Z INFO:conda_build.build:Packaging adorn-0.0.3-pyhd8ed1ab_0
2021-12-11T18:17:29.2274703Z INFO conda_build.build:bundle_conda(1529): Packaging adorn-0.0.3-pyhd8ed1ab_0
2021-12-11T18:17:29.2275967Z Packaging adorn-0.0.3-pyhd8ed1ab_0
2021-12-11T18:17:31.4953350Z number of files: 31
2021-12-11T18:19:36.5596416Z Unknown format
2021-12-11T18:19:36.5765783Z Unknown format
2021-12-11T18:19:36.5953041Z Unknown format
2021-12-11T18:19:36.6686755Z Unknown format
2021-12-11T18:19:36.7155172Z Unknown format
2021-12-11T18:19:36.7340477Z Unknown format
2021-12-11T18:19:36.7583394Z Unknown format
2021-12-11T18:19:36.7759921Z Unknown format
2021-12-11T18:19:36.7977521Z Unknown format
2021-12-11T18:19:36.8288259Z Unknown format
2021-12-11T18:19:36.8577407Z Unknown format
2021-12-11T18:19:36.9126524Z Unknown format
2021-12-11T18:19:36.9807914Z Unknown format
2021-12-11T18:19:37.0116551Z Unknown format
2021-12-11T18:19:37.0360625Z Unknown format
2021-12-11T18:19:37.0572663Z Unknown format
2021-12-11T18:19:37.3357732Z    INFO: sysroot: 'C:/Windows/' files: '['winhlp32.exe', 'win.ini', 'twain_32/wiatwain.ds', 'twain_32.dll']'
2021-12-11T18:19:37.3463121Z    INFO (adorn,Scripts/adorn.exe): Needed DSO C:/Windows/System32/KERNEL32.dll found in $SYSROOT
2021-12-11T18:19:37.3465392Z    INFO (adorn,Scripts/adorn.exe): Needed DSO C:/Windows/System32/SHLWAPI.dll found in $SYSROOT
2021-12-11T18:19:37.3529744Z    INFO (adorn): Interpreted package 'adorn' is interpreted by 'python'
2021-12-11T18:19:37.4961526Z [noarch_python] Noarch package contains binary script: adorn.exe
2021-12-11T18:19:43.1927018Z ##[error]Cmd.exe exited with code '1'.
2021-12-11T18:19:43.2755369Z ##[section]Finishing: Build recipe
```

Does anyone have any idea why this might be happening? The package is pretty straightforward.

File: recipes/adorn/LICENSE.rst [hunk 1]
```diff
@@ -0,0 +1,203 @@
+Apache License
```
Comment (dopplershift) [inline recipes/adorn/LICENSE.rst:1]: Since `LICENSE.rst` is already included in the PyPI tarball, you don't need to (and shouldn't) include it here.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Poetry still thinks you have a CLI:
```
[tool.poetry.scripts]
adorn = "adorn.__main__:main"
```
so it is creating a script that fails on Windows. You either need to add back in the entry point (and don't bother testing it) or patch `pyproject.toml` to remove it.

File: recipes/adorn/LICENSE.rst (hunk 1 above)
Comment (jacobBaumbach) [reply recipes/adorn/LICENSE.rst]: Thank you @dopplershift I have made both changes and the checks are now passing.

**Review by jacobBaumbach — COMMENTED** [review]
changed this as part of my commit

**Review by jacobBaumbach — APPROVED** [review]

**Review by dopplershift — APPROVED** [review]

**Pull request merged**